subrace/Dade F., Jonah M.; Dusk Elf.json

```diff
@@ -0,0 +1,148 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "YCoR",
+				"full": "Y's Codex of Realities",
+				"abbreviation": "YCoR",
+				"color": "782d99",
+				"authors": [
+					"Dade Fogleman",
+					"Jonah Mitchell"
+				],
+				"convertedBy": [
+					"RainerRaccoon"
+				],
+				"version": "1.0",
+				"url": "https://homebrewery.naturalcrit.com/share/mnjXuZQNgDBX",
+				"status": "ready",
+				"edition": "classic"

```

Comment:
these need to be pulled up a level, currently they're under `_meta.sources[0]` (rather than `_meta`)